packages/react-storage/src/components/StorageBrowser/views/LocationsView/LocationsView.tsx

```diff
@@ -102,34 +94,43 @@ export function LocationsView({
       hasError,
     });
 
+  // FIXME: Eventually comes from useView hook
+  const contextValue: ControlsContext = {
+    data: {
+      isDataRefreshDisabled: disableRefresh,
+    },
+    onDataRefresh: () => {
```

Comment:
Let's update to `onRefresh`, aligning the callback names will help to prevent confusion for future maintainers

Reply:
thx updated